conf/eslint-recommended.js

```diff
@@ -6,259 +6,14 @@
-        "no-ternary": "off",
-        "no-this-before-super": "error",
-        "no-throw-literal": "off",
-        "no-trailing-spaces": "off",
-        "no-undef": "error",
-        "no-undef-init": "off",
-        "no-undefined": "off",
-        "no-underscore-dangle": "off",
-        "no-unexpected-multiline": "error",
-        "no-unmodified-loop-condition": "off",
-        "no-unneeded-ternary": "off",
-        "no-unreachable": "error",
-        "no-unsafe-finally": "error",
-        "no-unsafe-negation": "error",
-        "no-unused-expressions": "off",
-        "no-unused-labels": "error",
-        "no-unused-vars": "error",
-        "no-use-before-define": "off",
-        "no-useless-call": "off",
-        "no-useless-computed-key": "off",
-        "no-useless-concat": "off",
-        "no-useless-constructor": "off",
-        "no-useless-escape": "off",
-        "no-useless-rename": "off",
-        "no-useless-return": "off",
-        "no-var": "off",
-        "no-void": "off",
-        "no-warning-comments": "off",
-        "no-whitespace-before-property": "off",
-        "no-with": "off",
-        "nonblock-statement-body-position": "off",
-        "object-curly-newline": "off",
-        "object-curly-spacing": ["off", "never"],
-        "object-property-newline": "off",
-        "object-shorthand": "off",
-        "one-var": "off",
-        "one-var-declaration-per-line": "off",
-        "operator-assignment": "off",
-        "operator-linebreak": "off",
-        "padded-blocks": "off",
-        "prefer-arrow-callback": "off",
-        "prefer-const": "off",
-        "prefer-destructuring": "off",
-        "prefer-numeric-literals": "off",
-        "prefer-promise-reject-errors": "off",
-        "prefer-reflect": "off",
-        "prefer-rest-params": "off",
-        "prefer-spread": "off",
-        "prefer-template": "off",
-        "quote-props": "off",
-        "quotes": "off",
-        "radix": "off",
-        "require-await": "off",
-        "require-jsdoc": "off",
-        "require-yield": "error",
-        "rest-spread-spacing": "off",
-        "semi": "off",
-        "semi-spacing": "off",
-        "sort-imports": "off",
-        "sort-keys": "off",
-        "sort-vars": "off",
-        "space-before-blocks": "off",
-        "space-before-function-paren": "off",
-        "space-in-parens": "off",
-        "space-infix-ops": "off",
-        "space-unary-ops": "off",
-        "spaced-comment": "off",
-        "strict": "off",
-        "symbol-description": "off",
-        "template-curly-spacing": "off",
-        "template-tag-spacing": "off",
-        "unicode-bom": "off",
-        "use-isnan": "error",
-        "valid-jsdoc": "off",
-        "valid-typeof": "error",
-        "vars-on-top": "off",
-        "wrap-iife": "off",
-        "wrap-regex": "off",
-        "yield-star-spacing": "off",
-        "yoda": "off"
-    }
-};
+module.exports = createFilteredConfig(rule => rule.meta.docs.recommended);
```

Comment:
The problem with this approach is that if we ever want to add a rule to eslint:recommended with non-default options, we have no mechanism to do that. We would have to add new config field to `meta`.

True. Since we don't have any rules like this right now, I don't consider this a big problem -- I think we can cross that bridge if we come to it.

> Also it's a bit weird to pull runtime configuration out of property marked as docs. The right way would be to move recommended out of the docs and into meta. But I think that's too much of a breaking change (although I have no idea who uses that, other then our demo).

I agree on both counts. Maybe we could move the property to `meta`, but add an additional property to `meta.docs` as part of the loading process, and remove the additional property in a future major version.